pkg/kube/client.go

```diff
@@ -18,8 +18,9 @@ import (
 	"sync"
 
 	"k8s.io/client-go/kubernetes"
-	// Empty import needed for GCP auth. See client-go documentation
+	// Empty imports needed for supported auth methods in kubeconfig. See client-go documentation
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
```

Comment:
I added this because locally we use oidc to auth to our clusters. I'm fine removing this change though and just putting it in locally when I want to test the UI and such.

Reply:
Neat - I don't see any reason to remove. I wonder if we should add it to our other projects.

Reply:
I actually found out recently that client-go can include all plugins with a single import. Let me know if y'all would like me to commit this instead.

```go
_ "k8s.io/client-go/plugin/pkg/client/auth"
```

Reference: https://github.com/kubernetes/client-go/blob/master/examples/out-of-cluster-client-configuration/main.go#L33-L40

Reply:
If that works, might as well do all the auth. Although, it's really designed to be run in-cluster, so it's a little weird to support all this external auth just for testing locally

Reply:
FWIW I've had issues in another project where importing the Azure auth broke my other k8s dependencies. If it works, go for it, just be aware it can potentially have unwanted side effects with deps.

Reply:
We can leave this as is or I'm also fine taking this line out and only putting it in when I need to test locally. Up to y'all really.

Reply:
Let's just leave it as-is